frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -1282,7 +1282,11 @@ dbg:guard(ReaderHighlight, "lookup",
     end)
 
 function ReaderHighlight:getSelectedWordContext(nb_words)
-    if not self.ui.rolling or not self.selected_text then return nil end
+    if not self.selected_text then return end
+    if not self.ui.rolling then
+        local _, prev_context, next_context = pcall(self.ui.document.getSelectedWordContext, self.ui.document, self.selected_text.text, self.selected_text.pos0, nb_words)
+        return prev_context, next_context
```

Comment:
In case pcall() fails, prev_context may be some error message - which probably should not be returned ?
(Not familiar with kopt to review its bit, so trusting you.)

Reply:
Thanks for pointing this out. Lua's a bit indirect sometimes.